src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -211,24 +212,52 @@
                 :func:`google.gax.construct_settings` for the structure of
                 this data. Falls back to the default config if not specified
                 or the specified config is missing data points.
-              app_name (string): The codename of the calling service.
-              app_version (string): The version of the calling service.
+              app_name (string): The name of the application calling
+                the service. Recommended for analytics purposes.
+              app_version (string): The version of the application calling
+                the service. Recommended for analytics purposes.
+              lib_name (string): The API library software used for calling
+                the service. (Unless you are writing an API client itself,
```

Comment:
Hm, to sound natural to my ears, "*self" always refers to the sentence subject. But definitely not something to block on.